packages/shared/src/services/ConfigService.ts

```
@@ -0,0 +1,460 @@
+          const env = currentConfig.app.environment;
+          
+          return {
+            environment: env,
+            isDevelopment: env === "development",
+            isProduction: env === "production",
+            isTest: env === "test",
+            isStaging: env === "staging",
+            debug: currentConfig.app.debug,
+            logLevel: currentConfig.app.logLevel
+          };
+        }),
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)